api-request-right-sizing-v2.md

```diff
@@ -68,29 +40,19 @@ Lower bound, in bytes, of the RAM recommendation. Defaults to 20MiB (20 * 1024 *
 {% swagger-parameter in="query" name="window" required="true" type="string" %}
 Required parameter. Duration of time over which to calculate usage. Supports days before the current time in the following format:
 
-`3d`. **Note**: Hourly windows are not currently supported. **Note**: It's recommended to provide a window greater than
-
-`2d`. See the
-
-[Allocation API documentation](allocation.md)
-
-for more a more detailed explanation of valid inputs to
-
-`window`.
+`3d`. **Note**: Hourly windows are not currently supported. **Note**: It's recommended to provide a window greater than `2d`. See the [Allocation API documentation](allocation.md) for more a more detailed explanation of valid inputs to `window`.
 {% endswagger-parameter %}
 
 {% swagger-parameter in="query" name="filter" type="string" required="false" %}
-A filter to reduce the set of workloads for which recommendations will be calculated. See [Filter parameters](filters-api.md)
-
-for syntax. V1 filters are also supported.
+A filter to reduce the set of workloads for which recommendations will be calculated. See [Filter parameters](filters-api.md) for syntax. V1 filters are also supported.
 {% endswagger-parameter %}
 
 {% swagger-parameter in="query" name="sortBy" type="string" %}
-Column to sort the response by. Defaults to "totalSavings". Options are "totalSavings", "currentEfficiency", "cpuRecommended", "cpuLatest", "memoryRecommended", and "memoryLatest".
+Column to sort the response by. Defaults to `totalSavings`. Options are `totalSavings`, `currentEfficiency`, `cpuRecommended`, `cpuLatest`, `memoryRecommended`, and `memoryLatest`.
 {% endswagger-parameter %}
 
 {% swagger-parameter in="query" name="sortByOrder" type="string" %}
-Order to sort by. Defaults to "descending". Options are "descending" and "ascending".
+Order to sort by. Default value `descending`. Options are `descending` and `ascending`.
```

Comment:
"Defaults to" is the verbiage that has been established thus far. Suggest continuing to use it for consistency.